frontend/src/main/java/puf/frisbee/frontend/view/GameView.java

```diff
@@ -98,12 +86,35 @@ private void handleKeyReleased(KeyEvent event) {
 	}
 	
 	@FXML
-	private void handleButtonGameOverContinueClicked(ActionEvent event) {
-		this.viewHandler.openWaitingView();
+	private void handleFrisbeeClicked(MouseEvent event) {
+		this.gameViewModel.incrementLabelScoreProperty();
+	}
+
+	@FXML
+	private void handleLevelContinueClicked(ActionEvent event) {
+		this.gameViewModel.setNextLevel();
+		this.gameViewModel.setLabelScoreProperty();
+		this.topPanelController.restoreRemainingTeamLives(this.gameViewModel.getTeamLives());
+		this.labelLevelSuccess.textProperty().bind(this.gameViewModel.getLabelLevelSuccessProperty());
+		this.buttonLevelContinue.textProperty().bind(this.gameViewModel.getButtonLevelContinueTextProperty());
+		this.gameViewModel.restart();
 	}
 	
+	@FXML
+	private void handleButtonGameOverContinueClicked(ActionEvent event) {
+		this.gameViewModel.resetLabelLevelProperty();
+		this.gameViewModel.resetLabelScoreProperty();
+		this.gameViewModel.resetTeamLives();
```

Comment:
Meinst du, es würde evtl. Sinn machen, hier nur eine `gameViewModel.continueGame()`Methode aufzurufen, die dann alles setzt? Dann haben wir weniger Logik in der View und mehr im ViewModel (und brauchen auch nicht soviele zusätzliche Methoden)?

Reply:
Hm ja, könnte man machen. Dann verschieben wir die Aufrufe ja nur … um es dann hier weniger transparent zu haben.